### PR TITLE
import and json fix

### PR DESCRIPTION
**Pull request by xithan**

### CODE REVIEW

File: mod/csv_import/set/type/file.rb [hunk 1]
```diff
@@ -1,3 +1,10 @@
 def csv?
-  attachment.content_type.in? ["text/csv", "text/comma-separated-values"]
+  attachment.content_type.in? %w[text/csv
+                                 text/comma-separated-values
+                                 text/plain
+                                 text/x-csv
+                                 text/x-comma-separated-values
+                                 application/vnd.ms-excel
```
Comment (ethn) [inline mod/csv_import/set/type/file.rb]: we can handle excel files?

File: mod/widgets/set/right/open_corporates.rb [hunk 2]
```diff
@@ -1,4 +1,13 @@
+def company_number
+  @company_number ||= content
+end
+
+def jurisdiction_code
+  @jurisdiction_code ||= left&.headquarters_jurisdiction_code
+end
+
 format :html do
+  delegate :company_number, :jurisdiction_code, to: :card
```
Comment (ethn) [inline mod/widgets/set/right/open_corporates.rb]: this is a pattern we should embrace.

I don't know if a `delegate_to_card` would be worth it, but maybe some shorter version?  `to_card`?

File: mod/csv_import/set/type/file.rb (hunk 1 above)
Comment (xithan) [reply mod/csv_import/set/type/file.rb]: no, but browsers on windows send csv files with that mime type

Comment (xithan) [reply mod/csv_import/set/type/file.rb]: vasso couldn't create import cards because of that

**Pull request merged**